h/services/mention.py

```diff
@@ -7,9 +7,8 @@
 from h.models import Annotation, Mention
 from h.services.html import parse_html_links
 from h.services.user import UserService
+from h.util.markdown_render import MENTION_ATTRIBUTE, MENTION_USERID
```

Comment:
It makes more sense that a domain-specific service depends on an external utility than the other way around, so I moved these constants there and imported them here.